### DISCUSSION TITLE
DRO for A-axis not showing

### DISCUSSION THREAD

**Discussion by FHUGO1** [winder/Universal-G-Code-Sender#1815] [Help]
Hi, 

I am running grbl mega 5x on UGS with a rotary A-axis. I was hoping the DRO would show the A -axis as G2core does. Is this an issue caused by selecting the GRBL firmware in UGS or GRBL mega not providing a status report for this axis?

I am running Platform 2.0.9

**Comment by breiler** [comment]
GRBL Mega is not yet supported. There is experimental support for it using Grbl_ESP32, FluidNC and g2core.
